vere/king.c

```diff
@@ -602,3 +607,26 @@ u3_king_commence()
   _king_loop_exit();
   exit(0);
 }
+
+/* u3_king_grab(): gc the kingdom
+*/
+void
+u3_king_grab(void* vod_p)
+{
+  //  XX fix leaks and enable
+  //
+#if 0
+  c3_w man_w = 0, pir_w = 0;
+  FILE* fil_u = stderr;
+
+  c3_assert( u3R == &(u3H->rod_u) );
+
+  fprintf(fil_u, "measuring king:\r\n");
+
+  man_w = u3m_mark(fil_u);
+  pir_w = u3_pier_mark(fil_u);
```

Comment:
Should this u3_pier_mark call be inside u3m_mark, or is there a reason to keep them separate?

Reply:
No, the various mark functions are concerned with their own roots (ie, areas of responsibility in the global state). Nothing in `noun` should depend on anything in `vere` (we're not quite there yet). Also, `u3m_mark` needs to be able to run in an inner road.

Reply:
Makes sense.